src/bin/lttng-sessiond/main.c

```diff
@@ -3444,10 +3557,18 @@ static int process_client_msg(struct command_ctx *cmd_ctx, int sock,
 	}
 	case LTTNG_ENABLE_EVENT:
 	{
+		struct lttng_event *ev = NULL;
 		struct lttng_event_exclusion *exclusion = NULL;
 		struct lttng_filter_bytecode *bytecode = NULL;
 		char *filter_expression = NULL;
 
+		ev = lttng_event_copy(&cmd_ctx->lsm->u.enable.event);
+		if (!ev) {
+			DBG("Failed to copy event");
```

Comment:
Log the name of the event here (`ev->name`, if I remember correctly).

Reply:
done.